oauthenticator/cilogon.py

```diff
@@ -180,16 +204,19 @@ def _validate_scope(self, proposal):
     def _validate_allowed_idps(self, proposal):
```

Comment:
Since using the value of `username_claim` as the default is now deprecated, we should validate that an `allowed_idps.username_derivation` always provides a username claim, I believe

Reply:
> we should validate that an allowed_idps.username_derivation always provides a username claim, I believe

The schema does this aready! `username_derivation` is required, and `username_claim` is required in it!

Reply:
Oh, totally forgot about that, sorry!